### ISSUE TITLE
define the response payload for the emoji endpoint

### ISSUE THREAD

**Issue by shiftkey** [octokit/routes#659]
- endpoint scope: Emoji
- endpoint name: Lists all the emojis available to use on GitHub
- route: GET /emoji
- documentation URL: https://developer.github.com/v3/emojis/#emojis

### Current specification

https://github.com/octokit/routes/blob/250db518150bc12a19155ed7dcfcccb82d35396e/openapi/api.github.com/operations/emojis/get.json#L21

### I expected

More details about the response payload.

I wanted to see if I could contribute this as a neat introduction, but it looks like this isn't supported in OpenAPI until version 3.1, based on the discussion here https://github.com/OAI/OpenAPI-Specification/issues/1505 - the relevant part of the JSON Schema spec is called [`patternProperties`](https://json-schema.org/understanding-json-schema/reference/object.html#pattern-properties) and I guess the schema would look like this:

```
"responses": {
  "200": {
    "description": "response",
    "content": {
      "application/json": {
        "schema": {
          "type": "object",
          "patternProperties": {
              "[a-z0-9+\\-]+": { "type": "string" }
          }
        }
    }
  }
},
```

I'm happy to take another shot at this when the schema supports OpenAPI 3.1 (I see it's 3.0.1 currently so maybe that's not far off):

https://github.com/octokit/routes/blob/250db518150bc12a19155ed7dcfcccb82d35396e/openapi/api.github.com/index.json#L2

**Comment by gr2m** [comment]
I don't think we can derive the response specification automatically from scraping [the endpoint's documentation](https://developer.github.com/v3/emojis/#emojis). But we can add a workaround for this particular endpoint in [lib/endpoint/overrides/workarounds.js](https://github.com/octokit/routes/blob/master/lib/endpoint/overrides/workarounds.js)
